### PR TITLE
Use es6 and modules

### PR DESCRIPTION
**Pull request by Fourteen98**
### New Features ###
Use ES6 syntax.
Divided code into modules
Refactored some methods to arrow functions.

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+// important to hold length of local  storage
+let id = 0;
+
+function getIndexInLocalStorage() {
+  const books = localS.getLocalStorage();
+  // eslint-disable-next-line no-undef
+  return books.length;
+}
```
Comment (Lordkaito) [inline js/index.js]: - [x] Please make sure to use ES6 arrow function syntax instead of the `function` keyboard.

File: js/index.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+const bookTitle = document.getElementById('title');
+const bookAuthor = document.getElementById('author');
+const form = document.getElementById('form');
+
+const localS = new LocalStorage();
+const ui = new CreateUI();
+
+function getVal(title, author, id) {
+  return new AwesomeBooks(title, author, id);
+}
```
Comment (Lordkaito) [inline js/index.js]: - [x] Just as mentioned below, please make sure to use arrow functions for all your functions.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @Fourteen98, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/modules/local_storage.js [hunk 3]
```diff
@@ -26,6 +31,12 @@ class LocalStorage {
 
     localStorage.setItem('books', JSON.stringify(books));
   }
+
+  setAfterWindowsLoad() {
+    const book = this.getLocalStorage();
+    bookTitle.value = book[book.length - 1].title;
+    bookAuthor.value = book[book.length - 1].author;
+  }
```
Comment (codepantha) [inline js/modules/local_storage.js]: - [x] The book list is broken after your recent changes. The book name and author doesn't show up after adding a new book. Please fix it 🙏

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUESTED CHANGES ♻️ 

## Hi @Fourteen98 👋, 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



### Highlights ⭐ 
✔️ Descriptive PR
✔️ Good commit history
✔️ Linter checks

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/modules/local_storage.js (hunk 3 above)
Comment (Fourteen98) [reply js/modules/local_storage.js:39]: @codepantha, can you kindly show me a short demonstration of where it doe not shows up.

**Review by zhadier — APPROVED** [review]
## Hi @Fourteen98  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**